### PR TITLE
podman: Add pasta to podman info

### PR DESCRIPTION
**Pull request by HirazawaUi**
[NO NEW TESTS NEEDED]

Fixes: #18561

#### Does this PR introduce a user-facing change?

```release-note
podman info will output information about the pasta if it exists in helper_binaries_dir or $PATH
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@Luap99 PTAL

File: libpod/networking_slirp4netns.go
Comment (Luap99) [inline libpod/networking_slirp4netns.go:67]: move the definition to networking_pasta_linux.go

File: libpod/info_linux.go
Comment (Luap99) [inline libpod/info_linux.go:74]: this line isn't necessary

**Review by Luap99 — COMMENTED** [review]
Thanks, some small comments and please also adjust the example output in https://github.com/containers/podman/blob/main/docs/source/markdown/podman-info.1.md to include the new output

**Comment by TomSweeneyRedHat** [comment]
LGTM in general, once the other comments are addressed.  Thanks @HirazawaUi !

**Comment by HirazawaUi** [comment]
> @HirazawaUi thank you for your PR, but why not spend a few seconds to add tests? Here's a simple one-liner that could prevent future regressions:
> 
> ```diff
> diff --git a/test/system/005-info.bats b/test/system/005-info.bats
> index f8a14ef1c..2f974439d 100644
> --- a/test/system/005-info.bats
> +++ b/test/system/005-info.bats
> @@ -43,6 +43,7 @@ host.conmon.package       | .*conmon.*
>  host.cgroupManager        | \\\(systemd\\\|cgroupfs\\\)
>  host.cgroupVersion        | v[12]
>  host.ociRuntime.path      | $expr_path
> +host.pasta                | .*executable.*package.*
>  store.configFile          | $expr_path
>  store.graphDriverName     | [a-z0-9]\\\+\\\$
>  store.graphRoot           | $expr_path
> ```

Thanks for reminder

File: docs/source/markdown/podman-info.1.md
Comment (edsantiago) [inline docs/source/markdown/podman-info.1.md:92]: Sigh. I'm really sorry, I really appreciate your updating the documentation.... but I think a casual reader might be confused by the mix of fc34/fc35 and amd64/aarch64 in this output. Could you massage it a little please? Change fc35 to fc34, and aarch64 to amd64 in all the new additions? (The output does not need to reflect True Reality™, it just needs to be internally consistent). Thank you!

Comment (HirazawaUi) [reply docs/source/markdown/podman-info.1.md:92]: > Sigh. I'm really sorry, I really appreciate your updating the documentation.... but I think a casual reader might be confused by the mix of fc34/fc35 and amd64/aarch64 in this output. Could you massage it a little please? Change fc35 to fc34, and aarch64 to amd64 in all the new additions? (The output does not need to reflect True Reality™, it just needs to be internally consistent). Thank you!

All right

**Comment by edsantiago** [comment]
LGTM! @containers/podman-maintainers PTAL.

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**
